pyuvdata/tests/test_utils.py

```diff
@@ -804,20 +803,20 @@ def test_mean_infs():
     out, wo = uvutils.mean_collapse(data, axis=0, return_weights=True)
     ans = np.arange(data.shape[1]).astype(np.float)
     ans[0] = np.inf
-    nt.assert_true(np.array_equal(out, ans))
+    assert np.array_equal(out, ans)
     ans = (data.shape[0] - 1) * np.ones(data.shape[1])
     ans[0] = 0
-    nt.assert_true(np.all(wo == ans))
+    assert np.all(wo == ans)
     print(data)
     out, wo = uvutils.mean_collapse(data, axis=1, return_weights=True)
     ans = np.mean(np.arange(data.shape[1])[1:]) * np.ones(data.shape[0])
     ans[0] = np.inf
     print(out)
     print(ans)
-    nt.assert_true(np.all(out == ans))
```

Comment:
looks like there are some stray print statements above this line, can we remove them?